R/meta_regression/treatment_comparison_page.R

```diff
@@ -49,6 +49,12 @@ covariate_treatment_comparisons_page_server <- function(
     output$package <- reactive({"gemtc"})
     outputOptions(x = output, name = "package", suspendWhenHidden = FALSE)
     
+    output$nodesplit_text <- renderText({
+      "This table only contains the estimates from the network meta analysis,
+        i.e. does not contain estimates from pairwise meta-analysis which only contains direct evidence.
+        If you would like to obtain the pairwise meta-analysis results, please run 4c-4. Nodesplit model"
+    })
```

Comment:
Since this text is not actually reactive, it can be put into the conditional panel directly, wrapping it in a `p()` tag